deployment/ansible/roles/cac-tripplanner.transitfeed/tasks/main.yml

```diff
@@ -1,4 +1,3 @@
 ---
 - name: Install transitfeed
-  pip: name=pytz
```

Comment:
This is an important optional dependency for some aspects of feed validation to work.